packages/output-handler/src/sqs-event-types.ts

```diff
@@ -3,15 +3,10 @@ import { stringToJSONSchema } from './zod-string-to-json';
 import { TranscriptionOutput } from '@guardian/transcription-service-common';
 
 const SQSMessageBody = z.object({
-	MessageId: z.string(),
-	Timestamp: z.string(),
-	Message: stringToJSONSchema.pipe(TranscriptionOutput),
+	messageId: z.string(),
+	body: stringToJSONSchema.pipe(TranscriptionOutput),
```

Comment:
non-blocking: I'm not sure I understand these changes. output-handler is still reading off of the same sqs queue, right? Why has the shape of the message changed?

Reply:
This was unexpected for me as well - I had to work out the structure of the new message by printing it out but I think think previously you'd have an sqs message wrapped around an sns notification, whereas now you've got an sqs message wrapped around the message itself.